tests/load-tests/run.sh

```diff
@@ -66,7 +66,6 @@ else
         -o "$output_dir" \
         -t "${THREADS:-1}" \
         $RANDOM_PREFIX_FLAG \
-        --disable-metrics \
```

Comment:
@sawood14012  Please make this configurable via an env variable with the `false` as default.

Something like:
```shell
--disable-metrics="${DISABLE_METRICS:-false}" \
```

Thanks

Reply:
okay